UserDetailsClient/UserDetailsClient.Core/Features/LogOn/B2CConstants.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace UserDetailsClient.Core.Features.LogOn
+{
+    public static class B2CConstants
```

Comment:
nit: consider reading these from a config file instead

Reply:
maybe future enhancement?

Reply:
i'll make an issue to track this...maybe someone will be interested and pick it up.

---
In reply to: [280920512](https://github.com/Azure-Samples/active-directory-b2c-xamarin-native/pull/85#discussion_r280920512) [](ancestors = 280920512)